file

```diff
@@ -28,7 +28,7 @@ See documentation on the [KOBMAN! website](https://kobman.github.io).
 
 Open your favourite terminal and enter the following:
 
-    $ curl -L https://raw.githubusercontent.com/kobman/KOBman/master/get.kobman.io | bash
+    $ curl -L https://raw.githubusercontent.com/asa1997/KOBman_target_repo/master/get.kobman.io | bash
```

Comment:
Should not have value corresponding to the namespace.

Reply:
So what should it be then?

Reply:
The pull request should not change the readme.md file.

Reply:
The value of the namespace changes when release.sh executes. Just like get.kobman.io.tmpl, README.md has a tmpl file under /scripts/tmpl README.md.tmpl. So when release.sh gets executed the @XXX@ variables are replaced in the readme file with the actual values using sed  command

Reply:
The readme.md need not be update in the upstream repo , hence the pull request should not have it. This is only used to test with your namespace , so can be avoided.

Reply:
OK. So can the readme be kept under /script/tmpl?

Reply:
you should ignore readme,md changes as it only for your repo.. It should not be part of the pull request.

Reply:
ok